src/CommonITILObject.php

```diff
@@ -6927,12 +6928,10 @@ class_exists($validation_class) && $params['with_validations']
                     continue;
                 }
 
-                $date = $document_item['date'] ?? $document_item['date_creation'];
-
                 $item = $document_obj->fields;
-                $item['date'] = $date;
+                $item['date'] = $document_item['date'] ?? $document_item['date_creation'];
```

Comment:
Don't you wan tto use `date_creation` if it exists?

Reply:
```suggestion
                $item['date'] = $document_item['date_creation'] ?? $document_item['date'];
```

Reply:
I guess the `date` field on the timeline entry should take in priority the `date` field of the corresponding item.